recipes/example/meta.yaml

```diff
@@ -42,6 +42,7 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
```

Comment:
Maybe add a comment noting that it is strongly encouraged (even if the license doesn't require it) to include the license file.